torchbenchmark/util/gitutils.py

```diff
@@ -0,0 +1,116 @@
+"""gitutils.py
+
+Utils for getting git-related information.
```

Comment:
might be worth using this rather than maintaining our own? 
https://gitpython.readthedocs.io/en/stable/tutorial.html